### PR TITLE
Make pkg_source_link more robust

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

File: R/report.R [hunk 1]
```diff
@@ -184,10 +184,8 @@ num_deps <- function(pkg) {
 }
 
 pkg_source_link <- function(chk) {
-  if (chk$cran) {
+  if (!is.null(chk$cran)) {
```
Comment (gaborcsardi) [inline R/report.R]: I think you also need to check of it is `TRUE`, so `isTRUE()` might be the best here.

Comment (hadley) [reply R/report.R]: I just tried that and it produced fewer good links than before

Comment (gaborcsardi) [reply R/report.R]: Interesting.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**